flutter_local_notifications/lib/src/platform_specifics/android/enums.dart

```diff
@@ -269,6 +269,48 @@ enum AudioAttributesUsage {
   final int value;
 }
 
+/// The available semantic actions for Android notifications.
```

Comment:
The API doc here doesn't explain what a semantic action is. Based on [this](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Action.SemanticAction), a better explanation would along the lines of "The available meanings that can be associated with a notification action."